### ISSUE TITLE
Appler via DOSBox/Retroarch

### ISSUE THREAD

**Issue by jgassie** [zajo/appler#8]
So I've been able to load Appler under RA/DOSBox-pure, but there are some odd things about arrow key or Joystick d-pad recognition after a DSK file has loaded.

If I go into the Disk Manager, such as with F3, the arrow keys or d-pad directional movements are recognized just fine.  Likewise, going into Keyboard mapping with F4, if I press down on the D-pad, the application recognizes that as a DOWN action, and prompts to re-assign the Down action.  But when I load a DSK object, the actual game rom is somehow not recognizing the kbd/joy events the way the game documentation says it should (arrow keys for example on the keyboard should work during the game, but they don't).

Do you have any idea what might be contributing to this issue, where the arrow keys are recognized via the menus, but not when running a game?   I will try using just normal dosbox under Windows to see if I encounter the same behavior, although I anticipate that it will be a similar experience.  I've not really encountered something similar with other RA+Dosbox apps so far.

Thank you for any additional information you can provide!

**Comment by jgassie** [comment]
OK, I couldn't get the arrow keys to work properly in regular dosbox either (when DSK is loaded).  So even though the arrow keys aren't being recognized, but should be supported by the game according to the manual, there are alternative keys that can be used A, Z, L, semicolon mentioned in the manual, instead of using arrow keys.  I think this will work.

**Comment by zajo** [comment]
Yeah, just tried to read the joystick with a for loop in basic using PDL(0), it doesn't seem to work.

**Comment by jgassie** [comment]
To workaround, I used the dbox core pad mapper under retroarch, and mapped joystick actions to perform the alternate keyboard keys, but was surprised the keyboard arrow keys aren't being recognized when the game is active with regular or RA dosbox. The arrow keys work in the appler menus though.

**Comment by jgassie** [comment]
I can't find any DSK file that doesn't seem to have the HGR problem with Appler, are you aware of some specific ones where this is not an issue?
